src/components/NavBar/NavbarMobile.jsx

```diff
@@ -57,9 +57,7 @@ const NavbarMobile = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
```

Comment:
Can now remove the `<Typography>` import too.

Reply:
I believe i deleted this. Just need to remove the css and do inline styling.